lib/src/configuration.dart

```diff
@@ -36,14 +36,23 @@ class Configuration {
   RealmSchema get schema => _schema;
 
   /// Creates a [Configuration] with schema objects for opening a [Realm].
+  ///
+  /// [fifoFilesFallbackPath] is a custom directory for storing FIFO special files
```

Comment:
JS has this doc API doc https://docs.mongodb.com/realm-sdks/js/latest/Realm.html#~Configuration

So something like: 
```suggestion
/// [fifoFilesFallbackPath] enables FIFO special files.
/// Opening a Realm creates a number of FIFO special files in order to coordinate access to the Realm across threads and processes. If the Realm file is stored in a location that does not allow the creation of FIFO special files (e.g. FAT32 filesystems), then the Realm cannot be opened. The FIFO special files are very lightweight and the main Realm file will still be stored in the location defined by the path property. Ignored if the filesystem supports FIFO special files. 
/// 
```

Reply:
OK I saw that this is documented below. So lets do this one instead.

Reply:
```suggestion
  /// [fifoFilesFallbackPath] enables FIFO special files. 
```